samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Adapter/Program.cs

```diff
@@ -22,6 +22,7 @@ static void Main()
             customization: transport =>
             {
                 transport.ConnectionString("host=localhost");
+                transport.DelayedDelivery().DisableTimeoutManager();
```

Comment:
docs snippets tend to avoid the fluent api variants eg https://docs.particular.net/transports/rabbitmq/delayed-delivery#backwards-compatibility-disabling-the-timeout-manager